### ISSUE TITLE
Show environment in dashboard (UI)

### ISSUE THREAD

**Issue by bforma** [bensheldon/good_job#1116]
I often have multiple browser tabs open with GoodJob Dashboard page from different Rails environments (ie. development and production). It would be great if I could somehow easily distinguish between pages from those different environments.

For example, show the environment label in the header (perhaps even differently colored per environment).

**Comment by bensheldon** [comment]
I have been thinking about this, especially after https://github.com/bensheldon/good_job/discussions/1109 was recently suggested.

I'm thinking that, once that secondary navbar exists, the content could be configured in the goodjob configuration e.g.

```ruby
# eval'ed into the partial
config.good_job.dashboard_message = -> { 
  "<strong>Environment:</strong> #{Rails.env}</strong>"
}
```

**Comment by bforma** [comment]
Yes, the secondary navbar would be a good place to put it 👍 . Why would this need extra custom configuration?

**Comment by bensheldon** [comment]
> Why would this need extra custom configuration?

I guess I'm proposing _not_ putting the environment string in it by default, but making it simple for you to insert it for your uses.
